src/main/java/ca/corefacility/bioinformatics/irida/service/impl/ProjectServiceImpl.java

```diff
@@ -266,7 +266,8 @@ public void removeUserFromProject(Project project, User user) throws ProjectWith
 	public void removeUserGroupFromProject(Project project, UserGroup userGroup) throws ProjectWithoutOwnerException {
 		final UserGroupProjectJoin j = ugpjRepository.findByProjectAndUserGroup(project, userGroup);
 		if (!allowRoleChange(project, j.getProjectRole())) {
-			throw new ProjectWithoutOwnerException("Removing this user group would leave the project without an owner.");
```

Comment:
There are a lot of random formatting changes to this file.  Make sure IntelliJ is set to format only VCS changes.  Don't worry about fixing this.

Reply:
Okies. Thank you for the screenshot.

Reply:
:) You're welcome.